onedal/decomposition/pca.py

```diff
@@ -59,36 +80,60 @@ def fit(self, X, queue):
         )
         covariance_matrix = from_table(cov_result.cov_matrix)
         self.mean_ = from_table(cov_result.means)
-        result = _backend.decomposition.dim_reduction.train(
+        pca_result = _backend.decomposition.dim_reduction.train(
             policy, params, to_table(covariance_matrix)
         )
 
-        self.n_components_ = self.n_components
-        self.variances_ = from_table(result.variances)
-        self.components_ = from_table(result.eigenvectors)
-        self.explained_variance_ = np.maximum(from_table(result.eigenvalues).ravel(), 0)
-        tot_var = covariance_matrix.trace()
-        self.explained_variance_ratio_ = self.explained_variance_ / tot_var
+        self.variances_ = from_table(pca_result.variances)
+        self.components_ = from_table(pca_result.eigenvectors)
+        self.explained_variance_ = np.maximum(
+            from_table(pca_result.eigenvalues).ravel(), 0
+        )
+        total_variance = covariance_matrix.trace()
         self.singular_values_ = np.sqrt((n_samples - 1) * self.explained_variance_)
```

Comment:
May be its better to align computation of singular_values on oneDAL side as well as explained_variance_